<launch>

    <node pkg="velodyne_height_map" type="heightmap_node" name="velodyne_height_map" output="screen">
    	<remap from="velodyne_points" to="/kitti/velo/pointcloud"/>

        <param name="cell_size" value="0.3"/>
        <param name="full_clouds" value="true"/>
        <param name="grid_dimensions" value="100"/>
        <param name="height_threshold" value="0.15"/>
    </node>

</launch>